config/locales/en.yml

```diff
@@ -21,3 +21,9 @@
 
 en:
   hello: "Hello world"
+  votes: {
```

Comment:
I knew you were forgetting the commas, but thought it better to for you to figure it out yourself. 

@Senjai quit being so helpful.